src/edu/csus/ecs/pc2/core/list/JudgementSortBySiteAcronymComparator.java

```diff
@@ -0,0 +1,34 @@
+package edu.csus.ecs.pc2.core.list;
+
+import java.io.Serializable;
+import java.util.Comparator;
+
+import edu.csus.ecs.pc2.core.model.Judgement;
+
+/**
+ * Sort judgemens by site, acronym, judgement text.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class JudgementSortBySiteAcronymComparator implements  Comparator<Judgement>, Serializable {
+
+    /**
+     * 
+     */
+    private static final long serialVersionUID = 1424848798314592182L;
+
+    @Override
+    public int compare(Judgement jOne, Judgement j2) {
+
+        if (jOne.getSiteNumber() != j2.getSiteNumber()) {
+            return jOne.getSiteNumber() - j2.getSiteNumber();
+        } else if (jOne.getAcronym().equals(j2.getAcronym())) {
+            return jOne.getDisplayName().compareTo(j2.getDisplayName());
+        } else {
+            return jOne.getAcronym().compareTo(jOne.getAcronym());
```

Comment:
This line of code makes no sense to me. Why is it comparing a Judgement Acronym to itself?

Reply:
copy/paste, typo, will push a fix.

Reply:
Pushed fix: 
commit b569c919d396e2d810b46c25700eba5649377eaf
Author: Douglas A. Lane <laned@ecs.csus.edu> 2022-01-25 09:49:15